plat/mediatek/mt8173/drivers/spm/spm_suspend.c

```diff
@@ -289,8 +319,48 @@ static enum wake_reason_t go_to_sleep_after_wfi(void)
 	return last_wr;
 }
 
+static void mt6311_control(unsigned int enable)
+{
+	unsigned int val;
+	unsigned int slave_addr = 0x6b;
+	unsigned int data_port = 0x8a;
```

Comment:
nit: These 2 vars could be const (or macros) but hopefully the compiler will optimise.